tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -2,6 +2,14 @@ import { checkResponseStatus, request } from '../http'
 import { Group, Me, User } from '../../types'
 import join from 'join-path'
 import { config } from '../../../config'
+import { getApplicationEntity } from './utils'
+
+enum Roles {
+  SpaceAdmin = '2aadd357-682c-406b-8874-293091995fdd',
+  Admin = '71881883-1768-46bd-a24d-a356a2afdf7f',
+  Guest = '38071a68-456a-4553-846a-fa67bf5596cc',
+  User = 'd7beeea8-8ff4-406b-8fb6-ab2dd81e6b11'
+}
```

Comment:
getApplicationEntity returns this information, maybe you could get role-ids from there instead of hardcoded.

Reply:
No idea whether these values will change or not
 
CC @ScharfViktor

Reply:
I think the IDs can be changed in the future. The best solution seems to me to get the roles from  GET `/graph/v1.0/applications`
<img width="819" alt="Screenshot 2023-02-08 at 09 25 06" src="https://user-images.githubusercontent.com/84779829/217475009-5e109df2-615d-40f7-8794-ad342f6bb306.png">

Reply:
^ @amrita-shrestha